src/ui/zcl_abapgit_html_form.clas.abap

```diff
@@ -484,6 +489,10 @@ CLASS ZCL_ABAPGIT_HTML_FORM IMPLEMENTATION.
         zcx_abapgit_exception=>raise( |Unexpected form field [{ <ls_entry>-k }]| ).
       ENDIF.
 
+      IF ls_field-required = abap_true AND <ls_entry>-v IS INITIAL.
+        zcx_abapgit_exception=>raise( |Unexpected empty form field [{ <ls_entry>-k }]| ).
+      ENDIF.
```

Comment:
This is OK for the default mechanism, but I'd suggest make it a bit less technical. Just `|{ here-is-full-field-name } field cannot be empty|`

Reply:
Exceptions show up in a banner. I removed this one.